.travis.yml

```diff
@@ -27,19 +27,22 @@ matrix:
       python: '3.7'
       install:
         - pip install -U pip setuptools "mypy~=0.740"
+        - pip install -U importlib-metadata~=5.0 # this needs to be installed separately BEFORE trying to parse the setup.py file
+        - pip install ".[mypy]"
       script:
         - python setup.py test --addopts "--cov-report term-missing"
-        - mypy .
+        - mypy . --exclude build --exclude sphinx_ext
     - stage: build
       python: '3.8'
       install:
         - pip install -U pip setuptools "mypy~=0.740"
+        - pip install ".[mypy]"
       script:
         - python setup.py test --addopts "--cov-report term-missing"
-        - mypy .
+        - mypy . --exclude build --exclude sphinx_ext
     - stage: deploy
       if: tag =~ ^[0-9]+\.[0-9]+\.[0-9]+
-      python: '3.6'
+      python: '3.7'
```

Comment:
The pymetrics build stage looked identical to this but ended up failing on python 3.6 and I had to make a second PR to bump to python 3.7. Making the same change here assuming that python 3.6 will fail to build this library as well.